index.js

```diff
@@ -1,9 +1,5 @@
 'use strict';
-var stringWidth = require('string-width');
+const stringWidth = require('string-width');
 
-module.exports = function (str) {
-	return Math.max.apply(null, str.split('\n').map(function (x) {
-		return stringWidth(x);
-	}));
-};
+module.exports = str => Math.max.apply(null, str.split('\n').map(stringWidth));
```

Comment:
Undo the `map(stringwidth)`, it's bad practice. The reason is that map actually calls stringwidth with more arguments, like the index. This can have a totally different outcome. Better to be explicit.

Reply:
Done.  For my future reference, can this actually make a difference when stringWidth only looks at a single parameter?  I realize it would obviously make a big difference with variadic functions.

Reply:
A second optional parameter can always be added as a minor upgrade which will cause this one to break.

Reply:
FYI https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-fn-reference-in-iterator.md

Not sure why XO didn't caught this one. On mobile, so can't check it.